pages/partials/questionScorePanel.ejs

```diff
@@ -31,8 +31,9 @@
       <% } %>
     </tbody>
   </table>
-  <% if (assessment.allow_issue_reporting) { %>
+  <% if (assessment.type == 'Exam' || assessment.allow_issue_reporting) { %>
```

Comment:
I think we want to trigger this with `authz_data.mode == 'Exam'`, which is true if we are in the CBTF IP range. People might be giving `Exam` type assessments outside of the CBTF, for example practice exams.

Reply:
Good point. I reworked the logic and tested it against HW and Exam questions in public and exam modes.